install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -825,6 +825,7 @@ data:
   enable-bgp-control-plane: "false"
 {{- end }}
 
+  procfs: {{ .Values.procfs | quote }}
```

Comment:
I would have expected that the user may want to choose where the _host_ procfs is, so that we can map that into the container. I don't understand why we would let the user configure the path _inside_ the container, since there's only one correct configuration parameter for this value (otherwise it's incompatible with the hardcoded changes in the `DaemonSet`).

Reply:
Oh, very good catch, it does not make sense to expose this option to the user. It should be this logic instead:
```
{{- if not .Values.securityContext.privileged }}
  procfs: "/host/proc"
{{- end }}
```